sinabs/backend/dynapcnn/chips/dynapcnn.py

```diff
@@ -174,65 +233,103 @@ def get_dynapcnn_layer_config_dict(cls, layer: DynapcnnLayer):
 
     @classmethod
     def write_dynapcnn_layer_config(
-        cls, layer: DynapcnnLayer, chip_layer: "CNNLayerConfig"
-    ):
+        cls,
+        layer: DynapcnnLayer,
+        layer2core_map: Dict[int, int],
+        destination_indices: List[int],
+        chip_layer: CNNLayerConfig,
```

Comment:
What is the difference between CNNLayerConfig and DVSLayerConfig?

Reply:
As the names were intended to suggest, CNNLayerConfig relates to the CNN (convolutional) layers of the chip, whereas DVSLayerConfig is for the sensor.